data.json

```
@@ -134,6 +135,7 @@
       "go1.21.13.linux-amd64.tar.gz",
       "go1.22.12.linux-amd64.tar.gz",
       "go1.23.6.linux-amd64.tar.gz",
+      "go1.24.6.linux-amd64.tar.gz",
```

I believe this is the cause of the strange `Fetching null` error in the CI run? 

```suggestion
      "go1.24.0.linux-amd64.tar.gz",
```

I wonder perhaps whether adding `--exit-status` to the jq usage here might bubble that error up sooner and make it easier to understand from the CI logs?
https://github.com/heroku/heroku-buildpack-go/blob/61b3a567c7a1d61617f50160f3a051616ddecfc7/sbin/fetch-test-assets#L41

xref https://jqlang.org/manual/#invoking-jq